hydpy/models/rconc_uh.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""Calculate runoff concentration by convolution of the isosceles triangular
+unit hydrograph.
+
+|rconc_uh| is a submodel that supplies its main model with the
+calculation of the runoff concentration by convolution of the isoscelese
+triangular unit hydrograph.
+
+The integration test of the application model |hland_v1| uses the
+|rconc_uh| submodel.
```

Comment:
Outdated. My suggestion:

|rconc_uh| is a submodel that supports its main model by calculating runoff
concentration using the unit hydrograph approach.  It allows for different unit
hydrograph shapes, which can be configured based on specific geometries or wholly
customised.  One example of a specific geometry is the isosceles triangle of HBV96 
:cite:p:`ref-Lindstrom1997HBV96`.  See the documentation on parameter |UH| for further 
information.  Also, see  the integration tests of application model  |hland_v1|, which 
use |rconc_uh| as a submodel.

Reply:
ok